BluePic-OpenWhisk/actions/Orchestrator.swift

```diff
@@ -8,45 +8,59 @@ import Foundation
 import SwiftyJSON
 
 func main(args: [String:Any]) -> [String:Any] {
-
+    
     var error:String = "";
     var returnValue:String = "";
-    let imageId: String? = String(describing: args["imageId"]!)
-    let targetNamespace: String? = args["targetNamespace"] as? String
-
-    let cloudantReadInvocation = Whisk.invoke(actionNamed: "/\(targetNamespace!)/bluepic/cloudantRead", withParameters: ["cloudantId": imageId!])
+    let imageId = args["imageId"] as? String ?? ""
+    let targetNamespace = args["targetNamespace"] as? String ?? ""
+    
+    let cloudantReadInvocation = Whisk.invoke(actionNamed: "/\(targetNamespace)/bluepic/cloudantRead", withParameters: ["cloudantId": imageId])
+    
     let response = cloudantReadInvocation["response"] as! [String:Any]
-    let payload = response["result"] as! [String:Any]
-    let documentString:String = payload["document"] as! String
-
-    let documentData = documentString.data(using: String.Encoding.utf8, allowLossyConversion: true)!
-    var document = JSON(data: documentData)
-
+    let documentPayload = response["result"] as! [String:Any]
```

Comment:
I think your best bet for avoiding force unwrapping is to just combine your `if let` statements because they all need to exist anyway for `document`, in this  case, to get data. Something like this:

        var document: JSON = [:]
        if let response = cloudantReadInvocation["response"] as? [String:Any],
        let documentPayload = response["result"] as? [String:Any],
        let documentString:String = documentPayload["document"] as? String,
            let documentData = documentString.data(using: String.Encoding.utf8, allowLossyConversion: true) {
            document = JSON(data: documentData)
        }